.github/workflows/builder_image_protobuf.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
   push:
     runs-on: ubuntu-22.04
     needs: build
-    if: github.ref_name == 'develop'
+    # if: github.ref_name == 'develop'
```

Comment:
Why? Revert or add note to comment.

Reply:
I'll revert before merging,  this image pipeline is far from optimal.

Reply:
Okey. Thanks